e2e/fromager_hooks/src/package_plugins/hooks.py

```diff
@@ -24,6 +24,22 @@ def after_build_wheel(
     test_file.write_text(f"{dist_name}=={dist_version}")
 
 
+def after_bootstrap(
+    ctx: context.WorkContext,
+    req: Requirement,
+    dist_name: str,
+    dist_version: str,
+    sdist_filename: pathlib.Path,
+    wheel_filename: pathlib.Path,
```

Comment:
The function signature does not match the signature of the hook:

```suggestion
    sdist_filename: pathlib.Path | None,
    wheel_filename: pathlib.Path | None,
```